Gemfile

```diff
@@ -36,6 +36,7 @@ gem "whenever", "1.0.0", require: false
 gem "with_advisory_lock", "~> 4.6"
 
 group :development do
+  gem "listen"
```

Comment:
Can you explain in the commit message why you've added this gem?

Reply:
This should be explained now 😄

Reply:
Great!